common/config/data.js

```diff
@@ -172,22 +176,90 @@ export type NodeConfig = {
   estimateGas: ?boolean
 };
 
+function makeEtherscanExplorer(url): BlockExplorerConfig {
```

Comment:
`makeEtherscanExplorer` seems like a nismoner. Perhaps `makeETHExplorer`?

Reply:
Well, it would only work with sites that have the same URL structure as Etherscan, so I didn't want anyone to throw in a totally different type of site and expect it to work. Though you're right that I do use it for gastracker.io, explorer.rsk.co, and gander.tec. They just all fortunately follow the same path spec.

Perhaps I'll rename, but add a comment to clarify.

Reply:
Oh, yeah, I was thinking this as well. It has to be a compatible url element path. A clarifying comment should be good 👍